assets/js/modules/analytics/components/common/ExistingTagNotice.js

```diff
@@ -31,126 +31,36 @@ import { MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
-					'google-site-kit'
-				),
-				ua.existingTag,
-				ga4.existingTag
-			);
-		} else if (
-			ua.existingTag !== ua.propertyID &&
-			ga4.existingTag === ga4.measurementID
-		) {
-			return sprintf(
-				/* translators: %1$s: Analytics tag ID, %2$s: Analytics 4 measurement ID */
-				__(
-					'An existing Universal Analytics tag with the ID %1$s and an existing Google Analytics 4 tag with the ID %2$s were found on your site. Since the Google Analytics 4 tag refers to the same property selected here, Site Kit will not place its own tag and rely on the existing one. If later on you decide to remove this tag, Site Kit can place a new tag for you.',
-					'google-site-kit'
-				),
-				ua.existingTag,
-				ga4.existingTag
-			);
-		}
-		return sprintf(
-			/* translators: %1$s: Analytics tag ID, %2$s: Analytics 4 measurement ID */
-			__(
-				'An existing Universal Analytics tag with the ID %1$s and an existing Google Analytics 4 tag with the ID %2$s were found on your site.',
-				'google-site-kit'
-			),
-			ua.existingTag,
-			ga4.existingTag
-		);
-	}
-
-	function getNoticeForExistingUATag() {
-		if ( ua.existingTag === ua.propertyID ) {
-			return sprintf(
-				/* translators: %s: Analytics tag ID */
-				__(
-					'An existing Universal Analytics tag was found on your site with the ID %s. Since this tag refers to the same property selected here, Site Kit will not place its own tag and rely on the existing one. If later on you decide to remove this tag, Site Kit can place a new tag for you.',
-					'google-site-kit'
-				),
-				ua.existingTag
-			);
-		}
-		return sprintf(
-			/* translators: %s: Analytics tag ID */
-			__(
-				'An existing Universal Analytics tag was found on your site with the ID %s.',
-				'google-site-kit'
-			),
-			ua.existingTag
-		);
-	}
-
-	function getNoticeForExistingGA4Tag() {
-		if ( ga4.existingTag === ga4.measurementID ) {
-			return sprintf(
-				/* translators: %s: Analytics 4 measurement ID */
-				__(
-					'An existing Google Analytics 4 tag was found on your site with the ID %s. Since this tag refers to the same property selected here, Site Kit will not place its own tag and rely on the existing one. If later on you decide to remove this tag, Site Kit can place a new tag for you.',
-					'google-site-kit'
-				),
-				ga4.existingTag
-			);
-		}
-		return sprintf(
-			/* translators: %s: Analytics 4 measurement ID */
-			__(
-				'An existing Google Analytics 4 tag was found on your site with the ID %s.',
-				'google-site-kit'
-			),
-			ga4.existingTag
-		);
+	if ( ! hasGTMAnalyticsProperty ) {
+		return null;
 	}
 
 	function getNoticeForExistingGTMPropertyAndGA4Tag() {
 		if (
-			gtmAnalyticsPropertyID === ua.propertyID &&
-			ga4.existingTag === ga4.measurementID
+			gtmAnalyticsPropertyID === propertyID &&
+			ga4ExistingTag === measurementID
```

Comment:
We don't need this function since it's intended for GTM and GA4 tags. We need to leave messages only for GTM tags which is implemented in the `getNoticeForExistingGTMProperty` function. See AC:

```
... the only check that should remain in the component is hasGTMAnalyticsProperty, and then it should display the notice from getNoticeForExistingGTMProperty().
```